x/fundraising/types/utils.go

```diff
@@ -34,8 +34,98 @@ func DeriveAddress(addressType AddressType, moduleName, name string) sdk.AccAddr
 
 // SortByBidPrice sorts bid array by bid price in descending order.
 func SortByBidPrice(bids []Bid) []Bid {
```

Comment:
Since the function sorts two things, is there a better name?

Reply:
I suggest just `SortBids`. We could use more verbose name like `SortBidsByPriceAndId` but later the sorting order could be changed, so I prefer simple name.